src/Select/web/Trigger.js

```diff
@@ -1,39 +1,14 @@
 import Flex from '../../Flex/web';
 
-const styles = {
-  borderColor(props) {
-    if (props.isOpen) {
-      return props.theme.color.primary;
-    } else if (props.disabled) {
-      return props.theme.color.greyLight;
-    } else if (props.error) {
-      return props.theme.color.red;
-    }
-    return props.theme.color.greyLight;
-  },
-  backgroundColor(props) {
-    if (props.isOpen) {
-      return props.theme.color.white;
-    } else if (props.disabled) {
-      return props.theme.color.greyLighter;
-    } else if (props.error) {
-      return props.theme.color.redLighter;
-    }
-    return props.theme.color.white;
-  },
-};
-
 const Trigger = Flex.extend`
```

Comment:
Although all of this works, how about a simpler fix like changing this line to
`const Trigger = styled.button``
and adding the respective focus styles to the Trigger component itself :)

Largely, I think the reason why {...getButtonProps()} isn't working in current master is that the Trigger element isn't a button, it's a Flex.

Reply:
Yeah, absolutely right.

I totally forgot that button element can render html after using all those UI libraries 😛 .